core/python_utils.py

```diff
@@ -34,7 +34,6 @@
     os.getcwd(), '..', 'oppia_tools', 'certifi-2021.5.30')
 sys.path.insert(0, _CERTIFI_PATH)
 
-import yaml  # isort:skip  pylint: disable=wrong-import-position, wrong-import-order
```

Comment:
Do we still need line 30? i.e, add YAML path to python path?

Reply:
@DiptoChakrabarty Reminder about this.

Reply:
okay done @YBCS

Reply:
@DiptoChakrabarty my bad, can you please  re-add line 29

Reply:
Sure